development/kops/set_environment.sh

```diff
@@ -70,4 +70,4 @@ export CNI_VERSION_URL=${ARTIFACT_URL}/plugins/${CNI_VERSION}/cni-plugins-linux-
 export CNI_ASSET_HASH_STRING=${CNI_ASSET_HASH_STRING:-sha256:$(curl -s ${CNI_VERSION_URL}.sha256 | cut -f1 -d' ')}
 export KOPS=${BASEDIR}/bin/kops-${RELEASE_BRANCH}
 mkdir -p ${BASEDIR}/bin
-export PATH=${BASEDIR}/bin:${PATH}
+export PATH=`pwd`/${BASEDIR}/bin:${PATH}
```

Comment:
Im not sure I understand why this would be neccessary?

Reply:
Since BASEDIR is not absolute, wanted to add the Absolute path to $PATH. We are going to store kops and kubectl in this dir.

Reply:
is it not absolute?

Reply:
No it is not. When we ran this using the top level makefile, it will have the value,
"development/kops".

Reply:
It is relative. When we run this using the top level makefile, it will have the value,
"development/kops".